Sasha signup login

amak-code opened the pull request:
Added input component to the SignUp and Login pages. Created tests for SIgnUp page. Added testId to the props in Input.tsx, so I could use it in other test files and change it's arguments.

jonathan-chin commented:
update Login page
add extra type tests to Input.test.tsx
update Signup.test.tsx

amak-code commented:
I merged chin-firebase-integration into my branch. Cleaned the code, implemented a login button with email and password. Added test for types to Input.test.tsx

jonathan-chin reviewed (approved):
looks good. will still need some cleaning on main
